app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class LikesController < ApplicationController
```

Comment:
The LikesController should also contain a `create` action. Some of the logic here should be separated into the create action.

This is to ensure that a users can like posts.